tests/testthat/test-add-subvariable.R

```diff
@@ -3,6 +3,25 @@ context('Adding subvariables')
 if (run.integration.tests) {
     with(test.authentication, {
         with(test.dataset(newDatasetFromFixture("apidocs")), {
+<<<<<<< HEAD
```

Comment:
Another merge conflict. I reworked the test we wrote a little bit, which is the part starting on L25. That should replace your L7-13, and I recommend adapting it to cover the new case you added of supplying a VarDef.